wiki/usage.md

```diff
@@ -0,0 +1,51 @@
+# Docker Usage Guide
+
+- [Docker Usage Guide](#docker-usage-guide)
+  * [Basic Usage](#basic-usage)
+  * [Docker Gotchas](#docker-gotchas)
+  * [Extending from the base image](#extending-from-the-base-image)
+  
+## Basic Usage
+
+To build and run `bats`' own tests:
+```bash
+$ git clone https://github.com/bats-core/bats-core.git
+Cloning into 'bats-core'...
+remote: Counting objects: 1222, done.
+remote: Compressing objects: 100% (53/53), done.
+remote: Total 1222 (delta 34), reused 55 (delta 21), pack-reused 1146
+Receiving objects: 100% (1222/1222), 327.28 KiB | 1.70 MiB/s, done.
+Resolving deltas: 100% (661/661), done.
+
+$ cd bats-core/
+$ docker build --tag bats:latest .
+...
+$ docker run -it bats:latest --tap /opt/bats/test
+```
+
+To mount your tests into the container, first built the image as above. Then:
+```bash
+$ docker run -v $(pwd)/test:/test bats /test
```

Comment:
As noted in #93, this should be something like:

```bash
docker run -it -v $(pwd):/opt/bats-dev bats:latest /opt/bats-dev/test
```

Reply:
I take this back slightly; the `/opt/bats-dev` dir should be `/opt/bats`, i.e.:

```
docker run -it -v $(pwd):/opt/bats bats:latest /opt/bats/test
```

And I'm suddenly finding this command extremely useful. 🙂